BangazonAPI/TestBangazonAPI/DatabaseFixture.cs

```diff
@@ -73,11 +91,14 @@ public void Dispose()
                 conn.Open();
                 using (SqlCommand cmd = conn.CreateCommand())
                 {
+                    cmd.CommandText = @$"DELETE FROM PaymentType WHERE AcctNumber='test'";
```

Comment:
There's really no reason to execute these twice, and it will slow your tests down incrementally. You could just keep adding to the command text string and then execute all at once.